src/esp/physics/bullet/BulletPhysicsManager.cpp

```diff
@@ -204,8 +188,8 @@ int BulletPhysicsManager::addArticulatedObjectFromURDF(
 
   // render visual shapes if either no skinned mesh is present or if the debug
   // flag is enabled
-  bool renderVisualShapes =
-      !renderAssetPath || model->getDebugRenderPrimitives();
+  bool renderVisualShapes = !renderAssetPath || (*renderAssetPath == "") ||
+                            model->getDebugRenderPrimitives();
```

Comment:
I know it didn't originate with this PR, but can we refactor away this "primitives" an "debug" verbiage throughout the API?
'\<visual\>' elements are not necessarily primitives or exclusively for debugging. I think this confuses things.
Something like "visual elements" or "link visual shapes" would be much better.